tez-dag/src/test/java/org/apache/tez/dag/app/launcher/TestContainerLauncherManager.java

```diff
@@ -514,26 +549,41 @@ public void stopContainer(ContainerStopRequest stopRequest) {
     }
   }
 
-  private static final String DAG_NAME = "dagName";
-  private static final int DAG_INDEX = 1;
-  public static class ContainerLauncherForTest extends ContainerLauncher {
+  public static class FailureReporterContainerLauncher extends ContainerLauncher {
 
-    public ContainerLauncherForTest(
-        ContainerLauncherContext containerLauncherContext) {
+    public FailureReporterContainerLauncher(ContainerLauncherContext containerLauncherContext) {
       super(containerLauncherContext);
     }
 
     @Override
-    public void launchContainer(ContainerLaunchRequest launchRequest) throws
-        ServicePluginException {
+    public void launchContainer(ContainerLaunchRequest launchRequest) throws ServicePluginException {
       getContext().reportError(ServicePluginErrorDefaults.INCONSISTENT_STATE, "ReportedFatalError", null);
     }
 
     @Override
     public void stopContainer(ContainerStopRequest stopRequest) throws ServicePluginException {
-      getContext()
-          .reportError(ServicePluginErrorDefaults.SERVICE_UNAVAILABLE, "ReportError", new DagInfoImplForTest(DAG_INDEX, DAG_NAME));
+      getContext().reportError(ServicePluginErrorDefaults.SERVICE_UNAVAILABLE, "ReportError",
+          new DagInfoImplForTest(DAG_INDEX, DAG_NAME));
     }
   }
 
+  /**
+   * This container launcher simply implements ContainerLauncher methods with the proper context callbacks.
+   */
+  public static class ContainerLauncherForTest extends ContainerLauncher {
```

Comment:
originally: ContainerLauncherForTest was a mock launcher for reporting errors, I simply renamed it to FailureReporterContainerLauncher, and turned ContainerLauncherForTest to a NO-OP0-ish container launcher with simply callbacks